rest-api/test/test-data/sites.csv

```diff
@@ -75,4 +75,8 @@ California ,University of California Irvine School of Medicine,7035987,HPO U of
 California ,University of Southern California,7035988,HPO USC,hpo-site-USC@prod.pmi-ops.org,CAL_PMC,
 Test Organization,Prod HPO Site A,,Prod HPO Site A,hpo-site-a@prod.pmi-ops.org,TEST,
 Pittsburgh,University of Pittsburgh - Mobile Team,7036372,HPO Pitt Mobile,hpo-site-pittmobile@prod.pmi-ops.org,PITT,
-Trans-American Consortium (TAC),Spectrum Health,7036220,HPO Spectrum,hpo-site-spectrum@prod.pmi-ops.org,TRANS_AM,
\ No newline at end of file
+Trans-American Consortium (TAC),Spectrum Health,7036220,HPO Spectrum,hpo-site-spectrum@prod.pmi-ops.org,TRANS_AM,

```

Comment:
Is this file supposed to stay up to date? For example, when we get tickets like https://precisionmedicineinitiative.atlassian.net/browse/DA-434 should there be a step to copy the CSV to this file too?

Reply:
Yes, it is... we should update this file as soon as we're using new sites in one of our cloud environments (with the intention of pushing to prod.)